docs/topics/modules/creating_new_application.adoc

```diff
@@ -48,7 +48,7 @@ ifeval::["{context}" == "user-guide"]
 +
 image::choose_mission_runtime.png[Choose mission and runtime]
 +
-. In the *Select Pipeline* section, select the appropriate option, then click the blue arrow to continue to the next step. The first option is suggested for most use cases because it provides stages to test your changes for each pipeline build. For more information see <<working_with_pipelines>>.
+. In the *Select Pipeline* section, select the appropriate option, then click the blue arrow to continue to the next step. The {osio} pipelines create repeatable, incrementally improving processes that test and deploy the code from creation to execution. The first option is suggested for most use cases. For more information see <<working_with_pipelines>>.
```

Comment:
The exact phrase again. Do we need it here? I suggest removing it in both places because it's just vague and not really telling us anything useful about this step where it's included.

Reply:
Exact phrase cause it is in the UG ifdef, whereas the previous one was for GS one. Basically I was trying to add a bit of context, but in this case probably not required since we have a link for more information.